README.md

```diff
@@ -4,4 +4,60 @@ This repository contains [Bazel](https://bazel.build/)
 [Starlark extensions](https://docs.bazel.build/versions/master/skylark/concepts.html)
 for defining fuzz tests in Bazel projects. 
 
-**This is not an officially supported Google product.**
\ No newline at end of file
+**This is not an officially supported Google product.**
+
+## Getting started
+
+To import rules_fuzzing in your project, you first need to add codes below to your `WORKSPACE` file:
+
+```python
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+        name = "rules_fuzzing",
+        sha256 = "8d98c444600ce25082b320a3364150806109fe33620a4e1c5bb856861591acb9",
+        strip_prefix = "bazel-rules-fuzzing-8520424714e827ff0360283e3237e35c8723d931",
```

Comment:
Just curious: Why do we need the `strip_prefix` attribute here?

Reply:
After checking https://docs.bazel.build/versions/master/repo/http.html#http_archive I think we can leave this option to the user.